app/views/dashboard/ng-partials/widgets/_saved_sequences.html.haml

```diff
@@ -1,14 +1,13 @@
 .widget-wrapper
   .row
     .small-12.columns
+      %button.green.button-like{"ng-click" => "addSequence()", style: "margin: 9px;" }
```

Comment:
Line is too long. [86/80]<br>Hash attribute should start with one space after the opening brace